src/map/map.h

```diff
@@ -716,6 +718,7 @@ struct map_data {
 		unsigned noknockback : 1;
 		unsigned notomb : 1;
 		unsigned nocashshop : 1;
+		unsigned noviewid : 22;
```

Comment:
Max Value is EQP_SHADOW_ACC_L = 0x200000, i.e 22 bit,
the : 22 means 22 Bit, and not value 22